Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -85,7 +85,7 @@ MenuBar::MenuBar(QWidget* parent) : QMenuBar(parent)
   AddHelpMenu();
 
   connect(&Settings::Instance(), &Settings::EmulationStateChanged, this,
-          [=](Core::State state) { OnEmulationStateChanged(state); });
+          [=, this](Core::State state) { OnEmulationStateChanged(state); });
```

Comment:
The lambda is unnecessary; we can replace it with `&MenuBar::OnEmulationStateChanged`.